components/director/internal/model/application.go

```diff
@@ -20,7 +20,7 @@ type Application struct {
 	ApplicationTemplateID *string
 	SystemNumber          *string
 	BaseURL               *string         `json:"baseUrl"`
-	Labels                json.RawMessage `json:"labels"`
+	OrdLabels             json.RawMessage `json:"labels"`
```

Comment:
Shouldn't we also update the `json` tag?

Reply:
On second thought, this comment would also be addressed for the DB entity... but there I guess the tag is tightly coupled to the DB column, so we can't rename the tag there without renaming the column. So it would be strange if the tag here is changed whereas the tag in the db entity stays the same.

I guess either rename the DB column, or just leave the tags as is.